public/app/features/canvas/runtime/element.tsx

```diff
@@ -40,12 +40,7 @@ export class ElementState implements LayerElement {
       horizontal: HorizontalConstraint.Left,
     };
     options.placement = options.placement ?? { width: 100, height: 100, top: 0, left: 0 };
-    this.validatePlacement();
-    this.sizeStyle = {
-      ...options.placement,
-      position: 'absolute',
-    };
-
+    this.updateLayout();
```

Comment:
🤔 is this necessary?  it will not actually do anything until the div exists, so I think we can just remove it